src/components/App/App.vue

```diff
@@ -32,35 +33,52 @@ export default {
   data: () => ({
     locations: [],
     cities: [],
-    states: []
+    states: [],
+    dropdownErrorMessage: ''
   }),
   mounted: function () {
     this.$nextTick(function () {
       this.retrieveLocalStorage()
-      fetchAPI.getStates().then(data => {
-        this.states = data.data
-      })
+      fetchAPI.getStates()
+        .then(data => {
+          this.states = data.data
+        })
+        .catch(() => {
+          this.dropdownErrorMessage = 'Please wait a minute and refresh the page. Something went wrong with the server.'
+        })
     })
   },
   methods: {
     retrieveCities (state) {
-      fetchAPI.getCities(state).then(data => {
-        this.cities = data.data
-      })
+      fetchAPI.getCities(state)
+        .then(data => {
+          this.cities = data.data
+        })
+        .catch(() => {
+          this.dropdownErrorMessage = 'Please wait a minute and refresh the page. Something went wrong with the server.'
+        })
     },
     retrieveLocale (location) {
-      fetchAPI.getLocale(location.city, location.state).then(value => {
-        value.data.id = Date.now()
-        this.checkExistingLocations(value.data)
-        this.updateLocalStorage()
-      })
+      fetchAPI.getLocale(location.city, location.state)
+        .then(value => {
+          value.data.id = Date.now()
+          this.checkExistingLocations(value.data)
+          this.updateLocalStorage()
+        })
+        .catch(() => {
+          this.dropdownErrorMessage = 'Please wait a minute and refresh the page. Something went wrong with the server.'
+        })
     },
     retrieveCurrent () {
-      fetchAPI.getCurrent().then(location => {
-        location.data.id = Date.now()
-        this.checkExistingLocations(location.data)
-        this.updateLocalStorage()
-      })
+      fetchAPI.getCurrent()
+        .then(location => {
+          location.data.id = Date.now()
+          this.checkExistingLocations(location.data)
+          this.updateLocalStorage()
+        })
+        .catch(() => {
+          this.dropdownErrorMessage = 'Please wait a minute and refresh the page. Something went wrong with the server.'
+        })
```

Comment:
If we're using this same error message in each of these catch statements, would it work / make sense to assign it to a variable to apply and DRY this up a little?